### ISSUE TITLE
Make Docker Container

### ISSUE THREAD

**Issue by ginccc** [aiden/autobot#2]
In order to use it in CI build environments and/or in scalable production envs, it would be awesome to have autobot within a docker container.

ideally it comes with the option of having a database as a second container (that can be launched via docker-compose) in order to store, maintain and update the test scripts

**Comment by lingz** [comment]
Yep this would be on our todo list also, should be really straight forward and basically just need to have a command to mount the test script directory into the container and then execute. This should make running autobot in your CI quite straightforward.

**Comment by ginccc** [comment]
cool! :-)
